src/test/java/com/aws/greengrass/lifecyclemanager/KernelTest.java

```diff
@@ -265,6 +265,7 @@ void GIVEN_kernel_WHEN_locate_finds_class_definition_in_config_THEN_create_servi
         try {
             kernel.parseArgs().launch();
         } catch (RuntimeException ignored) {
+            System.out.println("Error captured is: " + ignored.getMessage());
```

Comment:
Why do you need this? Use a logger instead?

Reply:
Oh did not mean to check this in. Was using it for some debugging.